Reply/app/src/main/res/values/styles.xml

```diff
@@ -36,4 +36,16 @@
         <item name="android:checkable">false</item>
     </style>
 
+    <style name="Widget.App.Button.OutlinedButton.IconOnly" parent="Widget.Material3.Button.ElevatedButton">
```

Comment:
You can get rid of this if you update to use a MaterialToolbar and a drawable that handles the navigation icon background